code/modules/clothing/masks/balaclava.dm

```diff
@@ -12,23 +12,33 @@
 	desc = "Designed to both hide identities and keep your face comfy and warm."
 	icon_state = "swatclava"
 	item_state = "balaclava"
-	flags_inv = HIDEFACE|BLOCKHAIR
-	w_class = ITEM_SIZE_SMALL
-	var/open = 0
+	action_button_name = "Adjust Balaclava"
+	var/open = 0 //0 = full, 1 = cover mouth, 2 = cover head
 
-/obj/item/clothing/mask/balaclava/tactical/proc/adjust_mask(mob/user)
-	if(!usr.incapacitated())
-		src.open = !src.open
-		if (src.open)
-			body_parts_covered = body_parts_covered & ~FACE
-			icon_state = initial(icon_state) + "_open"
-			user << "You open your face, putting the balaclava away."
-		else
-			gas_transfer_coefficient = initial(gas_transfer_coefficient)
-			body_parts_covered = initial(body_parts_covered)
-			icon_state = initial(icon_state)
-			user << "You pull balaclava up to cover your face."
-		update_wear_icon()
+/obj/item/clothing/mask/balaclava/tactical/proc/adjust_mask(mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	if(!user.incapacitated())
+		open = (open + 1) % 3
```

Comment:
...why?

Reply:
It's one line of code, instead of 3 lines that you would get if you manually set the state after the If's.

Reply:
Sorry, Drexample is right here.

While reducing unnecessary code is usually a plus, turning several readable lines of code into a confusing cipher is not good. Generally this sort of thing should only be done if necessary, for performance critical code (like lighting). Any human initiated action generally doesn't count as such

make this simpler and more readable please